orchard_poseidon/hash/fp.py

```diff
@@ -1,5 +1,9 @@
+#!/usr/bin/env python3
+import sys; assert sys.version_info[0] >= 3, "Python 3 required."
+sys.path.append('../..')
```

Comment:
```suggestion
sys.path.insert(1, os.path.join(sys.path[0], '../..'))
```
This is the recommended pattern because it works:
* even if the script was not invoked from the current directory;
* even if there are modules on `sys.path` that could accidentally include items with the same names.

(Found [here](https://stackoverflow.com/questions/714063/importing-modules-from-parent-folder#comment23054549_11158224).)